dotCMS/src/main/java/com/dotmarketing/factories/TreeFactory.java

```diff
@@ -1,8 +1,10 @@
 package com.dotmarketing.factories;
 
+import com.dotcms.util.transform.TransformerLocator;
 import com.dotmarketing.beans.Identifier;
 import com.dotmarketing.beans.Inode;
 import com.dotmarketing.beans.Tree;
+import com.dotmarketing.beans.transform.TreeTransformer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.beans.transform.TreeTransformer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)